pkgs/by-name/gm/gmic-qt/package.nix

```diff
@@ -107,24 +111,6 @@ stdenv.mkDerivation (finalAttrs: {
       gimp-plugin = gimpPlugins.gmic;
       inherit cimg gmic;
     };
-
-    updateScript = writeShellScript "gmic-qt-update-script" ''
```

Comment:
We should still have an update script because as-is nix-update and nixpkgs-update/r-ryantm will have a hard time finding updates both due to the lack of GitHub upstream releases (rather than just tags) and the fact that the tags have weird version strings. Maybe something like this?

```nix
    updateScript = nix-update-script {
      extraArgs = [ "--version-regex" "^v\\.(.*)" ];
    };
```

Also please remove the now-unused arguments from the top of the file

Reply:
Using gitUpdater would be nicer for this as it is specificly designd for git tags which this is using for versions.

```nix
    updateScript = gitUpdater { 
      rev-prefix = "v.";
    };
```

Reply:
Both that and nix-update-script are specifically designed for it I guess (nix-update just usually deals with edge cases better), but I'm largely indifferent as long as there is just *something* there :)

Reply:
Actually this is one of the edge cases that `gitUpdater` does not handle. If I add that `updateScript` then it tries to "update" it to version 2.2.0 rather than keeping the version at 3.3.5

So please use `nix-update-script` unless there is an argument for `gitUpdater` to fix that